endpoints/auth.go

```diff
@@ -562,7 +563,8 @@ func CurrentUser(c Context, scopes []string, audiences []string, clientIDs []str
 }
 
 func init() {
-	if appengine.IsDevAppServer() {
+	// TODO(jbd): Replace it with appengine.IsDevAppServer when it's implemented.
+	if os.Getenv("RUN_WITH_DEVAPPSERVER") != "" {
```

Comment:
But this doesn't change anything, does it? We're still using `appengine` package here, not `google.golang.org/appengine`.